include/cufinufft.h

```diff
@@ -19,10 +19,10 @@ int cufinufft_makeplan(int type, int dim, const int64_t *n_modes, int iflag, int
 int cufinufftf_makeplan(int type, int dim, const int64_t *n_modes, int iflag, int ntr,
                         float eps, cufinufftf_plan *d_plan_ptr, cufinufft_opts *opts);
 
-int cufinufft_setpts(cufinufft_plan d_plan, int M, double *d_x, double *d_y, double *d_z,
-                     int N, double *d_s, double *d_t, double *d_u);
-int cufinufftf_setpts(cufinufftf_plan d_plan, int M, float *d_x, float *d_y, float *d_z,
-                      int N, float *d_s, float *d_t, float *d_u);
+int cufinufft_setpts(cufinufft_plan d_plan, const int64_t M, double *d_x, double *d_y,
```

Comment:
matter of taste, but I usually don't `const` things that are passed by value. Is this how `finufft` does it?

Reply:
Huh. Yeah that's weird. Fixed.